### PR TITLE
add an example for kv test

### PR DESCRIPTION
**Pull request by vzqzhang**
## Change Description

- Add the kv test created for Dogfooding

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

File: examples/kv_tests/src/kvtest.cpp [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+    public:
+        using contract::contract;
+
+        [[eosio::action]]
+        void smalltest() {
+            require_auth(_self);
+            kvtest_table table{"kvtest"_n};
+
+	    uint64_t id = 1;
```
Comment (kimjh2005) [inline examples/kv_tests/src/kvtest.cpp:25]: Please check tab character here and below.

File: examples/kv_tests/src/kvtest.cpp [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+            uint64_t id = 1;
+            for (; id < 200; ++id)
+        	table.put({id, std::to_string(id) + data}, get_self());
+        	    
+            for (id = 1; id < 200; ++id) {
+                auto itr = table.id.find(id);
+                if (itr != table.id.end()) {
+        	    std::string val = itr.value().data;   
+        	    eosio::check(val.compare(data + std::to_string(id)), "The value for the key " + std::to_string(id) + " is " + val);
```
Comment (kimjh2005) [inline examples/kv_tests/src/kvtest.cpp:56]: val can be 8MB string too big to print.

File: examples/kv_tests/README.txt [hunk 3]
```diff
@@ -0,0 +1,16 @@
+--- kv_addr_book Project ---
```
Comment (kimjh2005) [inline examples/kv_tests/README.txt:1]: Wrong project name?

File: examples/kv_tests/src/kvtest.cpp [hunk 4]
```diff
@@ -0,0 +1,60 @@
+#include <eosio/eosio.hpp>
+#include <eosio/asset.hpp>
```
Comment (kimjh2005) [inline examples/kv_tests/src/kvtest.cpp:2]: Do you have to include this?

Comment (vzqzhang) [reply examples/kv_tests/src/kvtest.cpp:2]: Yes. I used eosio::check in the code.

**Review by kimjh2005 — APPROVED** [review]

**Comment by iamveritas** [comment]
A couple of observations:
1. since we already have in examples folder the kv_addr_book which is demonstrating the kv api, I think this could be merged into that one
2. however, if you think it makes more sense to be a stand alone example please give it a more suggestive name and comments in the code to be clear what is this example is trying to demonstrate.

**Pull request merged**